Oct news additions (#461)
LLNL/llnl.github.io: _posts/2020-09-30-ceed-ms35.md

---

> title: "CEED M-35 Milestone Report Released"

Reviewer: ```suggestion
title: "CEED MS-35 Milestone Report Released"
```

categories: release
---

> The Center for Efficient Exascale Discretizations ([CEED](https://github.com/CEED)) is part of the Department of Energy’s [Exascale Computing Project (ECP(https://www.exascaleproject.org/))]. The Center helps applications leverage future architectures by developing discretization algorithms that better exploit the hardware and deliver a significant performance gain over conventional methods. CEED regularly releases milestone (MS) [reports](https://ceed.exascaleproject.org/pubs/) to demonstrate progress and ensure transparency throughout its multi-year efforts.

Reviewer: ```suggestion
The Center for Efficient Exascale Discretizations ([CEED](https://github.com/CEED)) is part of the Department of Energy’s Exascale Computing Project ([ECP](https://www.exascaleproject.org/)). The Center helps applications leverage future architectures by developing discretization algorithms that better exploit the hardware and deliver a significant performance gain over conventional methods. CEED regularly releases milestone (MS) [reports](https://ceed.exascaleproject.org/pubs/) to document progress and ensure transparency throughout its multi-year efforts.
```

Author: New commit 92e9737

> The [MS-35 report](https://ceed.exascaleproject.org/docs/ceed-ms35-report.pdf) details the following goals:

Reviewer: ```suggestion
The [MS-35 report](https://ceed.exascaleproject.org/docs/ceed-ms35-report.pdf) details the CEED accomplishments towards the following goals:
```

Author: New commit 92e9737

- Work with vendors to port and run CEED benchmarks on early access systems for Aurora (Argonne National Lab) and Frontier (Oak Ridge National Lab)

> - Make the best BP/BK implementations from Nek, MFEM, libParanumal, and the external community available in [libCEED](https://github.com/CEED/libCEED)

Reviewer: ```suggestion
- Make the best CEED benchmark implementations from Nek5000, MFEM, libParanumal, and the external community available in [libCEED](https://github.com/CEED/libCEED)
```

Author: New commit 92e9737

- Organize the next CEED Annual meeting ([CEED4AM](https://ceed.exascaleproject.org/ceed4am/))

> - Optimize CEED applications and miniapps for Aurora and Frontier architectures

Reviewer: ```suggestion
- Optimize the CEED applications and miniapps for the Aurora and Frontier architectures
```

Author: New commit 92e9737

Author: New commit 1555bfe
